settings.gradle.kts

```diff
@@ -0,0 +1,20 @@
+pluginManagement {
+    repositories {
+        exclusiveContent {
+            forRepository {
+                maven {
+                    name = "GTNH Maven"
+                    setUrl("https://nexus.gtnewhorizons.com/repository/public/")
+                }
+            }
+            filter {
+                includeGroup("com.gtnewhorizons")
+                includeGroup("com.gtnewhorizons.retrofuturagradle")
+            }
+        }
+        gradlePluginPortal()
+    }
+}
+
+val minecraftVersion: String by settings
+rootProject.name = "jei-${minecraftVersion}"
```

Comment:
If you use IDEA, in my experience its best to set this like `rootProject.name = rootProject.projectDir.name` (I think that is valid .kts syntax) since sometimes the project structure can get corrupted otherwise

Reply:
Hmm ok. I use IDEA and this is the name I use in later JEI versions (where it's a gradle multiproject).
What kind of corruption issues did you see?